action.yaml

```diff
@@ -47,6 +47,13 @@ inputs:
       String to include in the cache key, in addition to the default, which is runner.os.
       This is useful when using multiple Go versions.
     required: false
+  path:
```

Comment:
Shouldn't we name this `working-directory`? `path` could easily be confused with the final argument of Staticcheck (the `./...`)

Reply:
The reason why I went with `path` was that other GitHub actions seem to name it this way. I took inspiration from https://github.com/actions/checkout. In the `README` they also call it `path`. Nonetheless, if this is not really relevant, I also think that `working-directory` is better.

Reply:
I don't think `actions/checkout` is applicable. There, `path` specifies where to put something, not where to execute.

I'm definitely interested in using what's most idiomatic. Alas, I am having a hard time finding comparable actions.

Reply:
Hm, good point. :+1:  Yeah, then let's just choose what's make the most sense. I'll change it to `working-directory` when I am home in 1-2 hours.

Reply:
Ok, I updated it. I also tested it again on my other projects and still works as expected.